tasks/modules/install-modules.yml

```diff
@@ -7,7 +7,7 @@
     - '"geoip" in nginx_modules'
     - nginx_install_epel_release | bool
 
-- name: Install NGINX modules
+- name: "Setup NGINX modules"
```

Comment:
Why is just this task name surrounded with double quotes?

Reply:
Good question, no real answer 😅